### DISCUSSION TITLE
Pterodactyl Official App

### DISCUSSION THREAD

**Discussion by VincyChannel** [pterodactyl/panel#3048] [Feature Requests]
Hi,
I would like if one day you would create an official pterodactyl panel application

**Comment by FoksVHox** [comment]
If you mean a desktop application, then I don't see a reason not to use a web-browser instead. If you mean a mobile application there is an app, however, it ain't compatible with 1.X.X

**Reply by VincyChannel** [reply]
Yes, I mean the mobile application, maybe I understand what app you are talking about, "Pterodactyl App" on Google Play Store, right?

**Reply by FoksVHox** [reply]
Yep

**Reply by VincyChannel** [reply]
Too bad it is not compatible with version 1.X.X

**Reply by trenutoo** [reply]
> Too bad it is not compatible with version 1.X.X

You have to wait until the new version for 1.x is released, it's being worked on as informed and pinned in the mobile channel on the Discord server

**Reply by VincyChannel** [reply]
Oh ok, then i will wait, I'll check on the discord server, thanks
@Software-Noob
